app/models/drug.rb

```diff
@@ -0,0 +1,37 @@
+class Drug < ActiveYaml::Base
+  set_root_path "config/data"
+  set_filename "drugs"
+
+  CREATED_TIME ||= File.ctime("config/data/drugs.yml").in_time_zone("UTC")
+  UPDATED_TIME ||= File.mtime("config/data/drugs.yml").in_time_zone("UTC")
+
+  CATEGORIES = {
+    hypertension_ccb: "Hypertension: CCB",
+    hypertension_arb: "Hypertension: ARB",
+    hypertension_diuretic: "Hypertension: Diuretic",
+    hypertension_ace: "Hypertension: ACE Inhibitor",
+    hypertension_other: "Hypertension: Other",
+    diabetes: "Diabetes",
+    other: "Other"
+  }.freeze
```

Comment:
An analogous enum exists in the ProtocolDrug class, but this class should probably contain the authoritative list.
Can make it an enum when this becomes an ActiveRecord backed class.